webapp/src/components/modals/create_issue/create_issue.jsx

```diff
@@ -105,6 +105,37 @@ export default class CreateIssueModal extends PureComponent {
 
     handleIssueDescriptionChange = (issueDescription) => this.setState({issueDescription});
 
+    renderIssueAttributeSelectors = () => {
+        if (this.state.repo && this.state.repo.permissions && !this.state.repo.permissions.push) {
+            return null;
+        }
```

Comment:
Does this `permissions.push` determine if they can create labels etc. specifically? I would think that means whether you can create issues/PRs?

Reply:
@mickmister It's documented [here](https://docs.github.com/en/rest/reference/issues#create-an-issue) that 
> Any user with pull access to a repository can create an issue.

Also from the same documentation:

![image](https://user-images.githubusercontent.com/15212283/90440903-ad1ee280-e0e0-11ea-9366-957dc31dc583.png)